win32/src/winapi/kernel32/init.rs

```diff
@@ -211,6 +211,8 @@ pub struct State {
     pub resources: pe::IMAGE_DATA_DIRECTORY,
     pub resource_handles: Handles<HRSRC, ResourceHandle>,
 
+    pub event_handles: Handles<HEVENT, EventHandle>,
```

Comment:
It looks like I was inconsistent about the _handles suffix between resource_handles vs files, whoops! No problem either way I guess.

Reply:
Yeah, I tried to fit in as much as possible. I believe that before I added `resource_handles` there was one with `_handles` and one without, but because `resources` was already taken I used the suffix. And that made it so that there was two `_handles` and only one without when I wrote this code 😄